Userland/Applications/Dictionary/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+serenity_component(
+    Dictionary
+    RECOMMENDED
+    TARGETS Dictionary
+    DEPENDS ImageDecoder RequestServer FileSystemAccessServer
```

Comment:
What is ImageDecoder and RequestServer used for?

Reply:
Probably nothing, this is all copied from TextEditor. I'll remove 'em.